auth0/v3/management/rules_configs.py

```diff
@@ -0,0 +1,62 @@
+from .rest import RestClient
+
+
+class RulesConfigs(object):
+
+    """RulesConfig endpoint implementation.
+
+    Args:
+        domain (str): Your Auth0 domain, e.g: 'username.auth0.com'
+
+        token (str): Management API v2 Token
+
+        telemetry (bool, optional): Enable or disable Telemetry
+            (defaults to True)
+    """
+
+    def __init__(self, domain, token, telemetry=True):
+        self.domain = domain
+        self.client = RestClient(jwt=token, telemetry=telemetry)
+
+    def _url(self, id=None):
+        url = 'https://%s/api/v2/rules-configs' % self.domain
+        if id is not None:
+            return url + '/' + id
+        return url
+
+    def all(self, stage='login_success', enabled=True, fields=None,
```

Comment:
Again, this is a lot of non-required params. We typically add fields and pagination as 1st-class and others to an `options` hash of some kind. This clues folks into how to reduce API call time and allows for expansion of additional filters in the future.